src/main/java/de/retest/recheck/cli/subcommands/Diff.java

```diff
@@ -22,8 +24,15 @@
 
 	@Override
 	public void run() {
-		// TODO Implement.
-		logger.info( "Not yet implemented!" );
+		try {
+			logger.info( "\n{}", ReplayResultLoader.load( testReport ) );
```

Comment:
This uses `ReplayResult#toString()` but didn't we say we want to stick to those printers for now?